01.w_Defects/buffer_underrun_dynamic.c

```diff
@@ -774,6 +774,7 @@ void dynamic_buffer_underrun_039()
 	{
 		for(i=-10;i<15;i++)
 		{
+                        // TODO: this loop does not use i so no buffer underrun actually occurs
```

Comment:
memset isn't actually going to overrun this buffer because it's just setting each of the bytes in the buffer 25 times.